src/openrct2/GameStateSnapshots.cpp

```diff
@@ -470,9 +469,9 @@ struct GameStateSnapshots final : public IGameStateSnapshots
     }
 
     void CompareSpriteDataGeneric(
```

Comment:
Rename to `CompareMiscEntityDataGeneric`

Reply:
Do you intend on dong it in this PR?